sdk/template/azure-sdk-template-two/CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+# Release History
+
+## 1.0.0-beta.1
```

Comment:
```suggestion
## 1.0.0-beta.1 (Unreleased)
```

Reply:
Good catch @alzimmermsft. I've added a commit that fixes these.